<launch>
  <node pkg="adaptive_clustering" name="adaptive_clustering" type="adaptive_clustering">
    <param name="print_fps"          value="true"/>
    <param name="leaf"               value="3"/>
    <param name="z_axis_min"         value="-0.8"/>
    <param name="z_axis_max"         value="2.0"/>
    <param name="radius_min"         value="0"/>
    <param name="radius_max"         value="30"/>
    <param name="cluster_size_min"   value="3"/>
    <param name="cluster_size_max"   value="2200000"/>
    <param name="k_merging_threshold"   value="0.2"/>
    <param name="z_merging_threshold"   value="0.01"/>
  </node>
  
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find adaptive_clustering)/launch/adaptive_clustering.rviz"/>
</launch>
